### PR TITLE
nerfs the t-42

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
nerfs the wield slowdown, the aim mode and a few other things

## Why It's Good For The Game
I think that we can all agree that the lmg is vastly too overpowered compared to it's rifle family sisters, and that it needs a severe nerf, so I have added way more wield slowdown and aim time.

## Changelog
:cl:
balance: t-42 nerfs
/:cl:

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
What type of reasoning is "we can all agree upon". LMG is a weapon of choice for many due to how comfortable and satisfying it is to use it. It also have a relatively good balanced mag capacity. Increasing it's wield delay would just turn it into a GPMG.

**Comment by Wayland-Smithy** [comment]
> Increasing it's wield delay would just turn it into a GPMG.

If wield delay is all they separates the two guns then maybe T42 has too much damage potential. For sure mag size. Good PR, but maybe not far enough, or maybe GPMG is too good all-around in comparison with stam buffs.

I'd still be slightly faster to wield, but have the matching aim-mode stats. Big deal. I would guess most players if not told it was changed would not even notice a difference until they run out of boolet 20 shots earlier.

**Comment by lovegreenstuff** [comment]
We cannot all agree that it's overpowered.

Mind explaining exactly how?

**Comment by Tristan63** [comment]
It already fires slower on aim mode than T12 or T18, and its ammo cannot be held on a standard belt. Not to mention attachment choices are rather poor.

**Comment by SplinterGP** [comment]
OP how?

**Comment by Wayland-Smithy** [comment]
> It already fires slower on aim mode than T12 or T18, and its ammo cannot be held on a standard belt. Not to mention attachment choices are rather poor.

If it is such a bad gun how come it makes up half the armament of unga balls?

**Comment by DerrickChamberlain** [comment]
Slowdown feels fine, weapon feels fine, has half the capacity of a GPMG, so I am not sure what is wrong with it. Guess this gun will be braziled

**Comment by DerrickChamberlain** [comment]
> OP how?

Yeah I am not sure how it is more OP then other options we have. It is a No from me.

**Comment by DerrickChamberlain** [comment]
> > It already fires slower on aim mode than T12 or T18, and its ammo cannot be held on a standard belt. Not to mention attachment choices are rather poor.
> 
> If it is such a bad gun how come it makes up half the armament of unga balls?

Not sure, personally I see far greater use of GPMG with LMGs then I do with T-42. I see less T-42's floating about, but maybe I am just missing something here.

File: code/modules/projectiles/magazines/rifles.dm [hunk 1]
```diff
@@ -206,7 +206,7 @@
 	caliber = CALIBER_10X24_CASELESS
 	default_ammo = /datum/ammo/bullet/rifle
 	w_class = WEIGHT_CLASS_NORMAL
-	max_rounds = 120
+	max_rounds = 100
```
Comment (DerrickChamberlain) [inline code/modules/projectiles/magazines/rifles.dm:209]: Really?! Even less ammo? That is ridiculous, slowly ROF and less ammo.

**Comment by SplinterGP** [comment]
> 
> 
> > It already fires slower on aim mode than T12 or T18, and its ammo cannot be held on a standard belt. Not to mention attachment choices are rather poor.
> 
> If it is such a bad gun how come it makes up half the armament of unga balls?

because people like, last i checked a gun being alot used  doens't mean its bad,

**Comment by lovegreenstuff** [comment]
> If it is such a bad gun how come it makes up half the armament of unga balls?

Because a lot of people prefer comfort to actual effectiveness. T-60 sucks fucking dick but people use it all the time because they barely have to reload.

**Comment by Losenis** [comment]
> If it is such a bad gun how come it makes up half the armament of unga balls?

Because 120 ammo means less reloading, which is a pita when you have to either:
A) Unwield, switch hands, click container and pull magazine out (or right click it), click on the gun with the nonzero chance of clicking the slot rather than the gun accidentally switching hands, switching hands again, wielding again
B) Clickdrag magazine to gun which you won't actually figure out on your own unless you're specifically told about it or RNGesus gives out the tip on roundstart while you happen to be on, needing to pixelhunt both magazine and gun.
And all of the above which gets 100x times worse in the middle of combat which can be interrupted with knockdowns or stuns making you start from even less than scratch due to having to pull out the gun again.

It's why you have GPMGers despite the gun being FF central due to having scatter even while staying still unless you specifically packed bipod that locks you in place. You don't see a lot of people running T12 and T18 due to the aim mode speed because they're awful to reload, even moreso the latter. It doesn't help that the gun covers part of the backpack, so you can accidentally right click the gun instead of the satchel to take a gun out and end up toggling safety on the gun.

**Comment by Tristan63** [comment]
> > It already fires slower on aim mode than T12 or T18, and its ammo cannot be held on a standard belt. Not to mention attachment choices are rather poor.
> 
> If it is such a bad gun how come it makes up half the armament of unga balls?

Because due to its ammo capacity, and certain attachments, it can be a good solo rusher weapon. You barely have to reload in a rush, and you can kill 2-3 xenos then reload when the advance catches up to you. Its fire rate off of aim mode is decent for what it is, and it also isn't as cumbersome as the T60 which isn't a viable rush weapon.

**Comment by davidchan13** [comment]
GPMG is far more common and LMG is used by a niche group of people who decip squad back in the day. It's only on paper that the T-42 is superior to a T-12 or T-60, in practical use the xeno has to cooperate with you (or be absolutely fucking stupid) to get that maximum potential of damage and mobility the T-42 supposedly has. You're too slow to chase, you don't have near as much suppression as a T-60 and the only real use for the T-42 today is to walk fire into a xeno position to make them scatter, something any defiler, boiler or queen can break in a heartbeat.

**Comment by TiviPlus** [comment]
literally why though

**Comment by TiviPlus** [comment]
Unneeded nerf splinter agreed

**Pull request closed without merging**